[Ingest Manager] Use docker registry for fleet api integration tests (#72621)
elastic/kibana: x-pack/plugins/ingest_manager/README.md

This plugin follows the `common`, `server`, `public` structure from the [Architecture Style Guide
](https://github.com/elastic/kibana/blob/master/style_guides/architecture_style_guide.md#file-and-folder-structure). We also follow the pattern of developing feature branches under your personal fork of Kibana.

### Tests

#### API integration tests

You need to have `docker` to run ingest manager api integration tests

> 1. In one terminal, change to the `x-pack` directory and start the test server with

Reviewer: ```suggestion
1. In one terminal, run the tests from the Kibana root directory with
```

   ```
   INGEST_MANAGEMENT_PACKAGE_REGISTRY_PORT=12345 yarn test:ftr:server --config x-pack/test/ingest_manager_api_integration/config.ts
   ```

1. in a second terminal, run the tests from the Kibana root directory with

   ```
   INGEST_MANAGEMENT_PACKAGE_REGISTRY_PORT=12345 yarn test:ftr:runner --config x-pack/test/ingest_manager_api_integration/config.ts
   ```

>    Optionnaly you can filter which tests you want to run using `--grep`

Reviewer: ```suggestion
   Optionally you can filter which tests you want to run using `--grep`
```
